js/src/modal.js

```diff
@@ -127,8 +127,6 @@ class Modal {
 
     this._adjustDialog()
 
-    $(document.body).addClass(ClassName.OPEN)
```

Comment:
does it works to move this line abode `this._adjustDialog()` ?

Reply:
I tried that but didn't work. We need to reduce the number of renderings.